File: trabajo 7/metodo_burbuja.py
def burbuja(arr):
    n = len(arr)
    for i in range(n):
        for j in range(n - 1 - i):
            if arr[j] > arr[j + 1]:
                
                arr[j], arr[j + 1] = arr[j + 1], arr[j]
    return arr

lista = [5, 3, 8, 4, 2]
ordenada = burbuja(lista)
print("Lista desordenada",lista)
print("Lista ordenada:", ordenada)

File: trabajo 7/metodo_burbuja.cpp
#include <iostream>
using namespace std;

void burbuja(int arr[], int n) {
    int temp;
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n - 1 - i; j++) {
            if (arr[j] > arr[j + 1]) {
                temp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = temp;
            }
        }
    }
}


void imprimirArreglo(int arr[], int n) {
    for (int i = 0; i < n; i++) {
        cout << arr[i] << " ";
    }
    cout << endl;
}


int main() {
    int lista[] = {5, 3, 8, 4, 2};
    int n = sizeof(lista) / sizeof(lista[0]);

    cout << "Lista desordenada: ";
    imprimirArreglo(lista, n);

    burbuja(lista, n);

    cout << "Lista ordenada: ";
    imprimirArreglo(lista, n);

    return 0;
}
